Source/iOS/Device.swift

```diff
@@ -44,6 +44,7 @@ open class Device {
             case "iPad3,1", "iPad3,2", "iPad3,3":            return Version.iPad3
             case "iPad3,4", "iPad3,5", "iPad3,6":            return Version.iPad4
             case "iPad6,11", "iPad6,12":                     return Version.iPad5
+            case "iPad7,5", "iPad7,6":                       return Version.iPad5
```

Comment:
Should it be `return Version.iPad6` here?

Reply:
wups